internal/server/v1/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,22 @@
+package kubernetes
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	entropyv1beta1 "go.buf.build/odpf/gwv/odpf/proton/odpf/entropy/v1beta1"
+	shieldv1beta1 "go.buf.build/odpf/gwv/odpf/proton/odpf/shield/v1beta1"
+)
+
+const (
+	pathParamProjectSlug = "projectSlug"
+	kindKubernetes       = "kubernetes"
+
+	// shield header names.
+	// Refer https://github.com/odpf/shield
+	headerProjectID = "X-Shield-Project"
```

Comment:
Is this intended to not being configurable?

Reply:
Yes, we are planning to get rid of dependency on header by using the project id sent in URL. As of now there is a limitation from shield side. Once that is fixed we will get rid of this. Hence didn't extract this out.